.github/workflows/release_draft.yml

```diff
@@ -51,19 +51,16 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           tag-name: ${{ github.event.inputs.tag-name }}
-          chapters: '[
-            {"title": "No entry 🚫", "label": "duplicate"},
-            {"title": "No entry 🚫", "label": "invalid"},
-            {"title": "No entry 🚫", "label": "wontfix"},
-            {"title": "No entry 🚫", "label": "no RN"},
-            {"title": "Breaking Changes 💥", "label": "breaking-change"},
-            {"title": "New Features 🎉", "label": "enhancement"},
-            {"title": "New Features 🎉", "label": "feature"},
-            {"title": "Bugfixes 🛠", "label": "bug"},
-            {"title": "Infrastructure ⚙️", "label": "infrastructure"},
-            {"title": "Silent-live 🤫", "label": "silent-live"},
-            {"title": "Documentation 📜", "label": "documentation"}
-          ]'
+          chapters: |
+            - { title: No entry 🚫, label: duplicate }
+            - { title: Breaking Changes 💥, label: breaking-change }
+            - { title: New Features 🎉, label: enhancement }
+            - { title: New Features 🎉, label: feature }
+            - { title: Bugfixes 🛠, label: bug }
+            - { title: Infrastructure ⚙️, label: infrastructure }
+            - { title: Silent-live 🤫, label: silent-live }
+            - { title: Documentation 📜, label: documentation }
```

Comment:
Here you use different pattern than in README. Please see the comment

Reply:
You are right. String, number, and boolean inputs are the only supported inputs here. The `|` was missing from the README usage examples.